ui/app/AppLayouts/Onboarding/popups/UploadProfilePicModal.qml

```diff
@@ -89,7 +89,8 @@ ModalPopup {
                     qsTrId("image-files----jpg---jpeg---png-")
                 ]
                 onAccepted: {
-                    selectedImage = imageDialog.fileUrls[0];
+                    cropImageModal.selectedImage = imageDialog.fileUrls[0];
```

Comment:
I'm thinking that if users cancel's the file open dialog `fileUrls` is empty. Will a guard work good enough?
```qml
                    if(imageDialog.fileUrls.length > 0) {
                        cropImageModal.selectedImage = imageDialog.fileUrls[0];
                        cropImageModal.open()
                    }
```

Reply:
The accepted() is emitted only when the file is chose. However, I added the guard to be double sure :)